armi/reactor/assemblies.py

```diff
@@ -43,6 +43,7 @@ class Assembly(composites.Composite):
     """
 
     pDefs = assemblyParameters.getAssemblyParameterDefinitions()
+    blockType = None
```

Comment:
Two suggestions:

1. Make this more clearly a non-public, class-specific variable that people should not try to edit / access. Something like `_BLOCK_TYPE`. We can add a [`ClassVar`](https://docs.python.org/3/library/typing.html#typing.ClassVar) type hint to further discourage instances modifying this type.
2. If it's public, can we add a docstring?

Reply:
I did (1), good call.